docs/release-notes/master.adoc

```diff
@@ -17,6 +17,15 @@ include::topics/making-open-source-more-inclusive.adoc[]
 == Introduction
 {ProductFullName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on {ocp-first}. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to {ocp-short} through the user interface (UI) and command-line interface (CLI).
 
+[id="mta-7-1-1"]
+== {ProductShortName} 7.1.1
+
+include::topics/ref_known-issues-7-1-1.adoc[leveloffset=+2]
+
+include::topics/ref_resolved-issues-7-1-1.adoc[leveloffset=+2]
+
+:!release-notes:
```

Comment:
this is bugging me but i know it is really pedantic... why are we closing the `release-notes` variable here and not at the end of the file? it will not impact the docs but it is weird. LOL

Reply:
No, you are right, I just missed deleting it. That's a good catch. I'll fix it later, along with other final changes.